### PR TITLE
website: Update Mac OS getting-started content

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### CODE REVIEW

File: website/docs/Installation/macos-install.md [hunk 1]
```diff
@@ -5,8 +5,6 @@ sidebar_position: 3
 # MacOS
 ## Installing Podman Desktop on MacOS using .dmg file
 
-#### Prerequisite: Podman Machine
-
 If you already have a Podman machine up and running, you can check out the [Downloads](/downloads/macos) section of this website to get the applicable .dmg file depending on your Mac Hardware Architecture (i.e. Intel or Apple M1).
```
Comment (slemeur) [inline website/docs/Installation/macos-install.md]: That's the first thing the reader will read.
If I'm new to podman, I'd have no idea what a "podman machine" is.

File: website/docs/Installation/macos-install.md [hunk 2]
```diff
@@ -5,8 +5,6 @@ sidebar_position: 3
...
 
 If you are new to Podman and do not have a Podman Machine up and running, fret not. Downloading and Installing Podman Desktop allows you to get all the dependencies as well.
```
Comment (slemeur) [inline website/docs/Installation/macos-install.md]: why do we even mention "podman machine" here?

File: website/docs/Installation/macos-install.md [hunk 3]
```diff
@@ -5,8 +5,6 @@ sidebar_position: 3
 # MacOS
 ## Installing Podman Desktop on MacOS using .dmg file
```
Comment (slemeur) [inline website/docs/Installation/macos-install.md:15]: I think this section needs to be re-wrote. 
Should be clear and guiding. Maybe move to a bullet list with clear instruction.
Also explain what happens when starting podman desktop for the first time. There are the pre-flight checks + what do we do when initializing.
Also important - once installed and setup, there should be CAT on what to do next!

**Review by slemeur — CHANGES_REQUESTED** [review]

File: website/docs/Installation/macos-install.md (hunk 1 above)
Comment (slemeur) [reply website/docs/Installation/macos-install.md]: Then, there are multiple binaries, the document should explain to the reader how to chose the right one. The universal binary being here to simplify things, so the user does not have to worry/know about which architecture they have (--> should be the default/recommended path).

File: website/docs/Installation/macos-install.md (hunk 3 above)
Comment (deekay2310) [reply website/docs/Installation/macos-install.md:15]: CAT?

**Comment by deekay2310** [comment]
@slemeur have added about view detection checks and removed details about the 'machine'

**Comment by deekay2310** [comment]
@benoitf can we merge?

**Review by slemeur — APPROVED** [review]
LGTM

**Pull request merged**